custom_components/opensprinkler/sensor.py

```diff
@@ -147,7 +147,7 @@ def unique_id(self) -> str:
     @property
     def unit_of_measurement(self):
         """Return the unit of the flow rate."""
-        return "liters/pulse"
+        return "L/min"
```

Comment:
should this be `L` or `l`?

Reply:
how about `ℓ` instead?

Reply:
Others use L for liters so would agree with "L/min".